README.md

```diff
@@ -1,22 +1,89 @@
-How to use archapp:
+archapp
+=======
 
-Create a conda environment that uses python 3. Make sure xarray is installed in the conda environment.
+PCDS conda Python environments already include archapp.
+
+In order to access the archiver interactively, use the following:
 
-Go to archapp/lib and type ipython in the command line.
+Source the environment first:
+```bash
+$ source /cds/group/pcds/pyps/conda/pcds_conda
+```
+
+And then start IPython:
+```bash
+$ ipython
+```
+
+Import a helper class that lets you access the archiver:
 
 ```python
-#Type: 
 from archapp.interactive import EpicsArchive
+```
 
-#Now create an object ---> 
+Now create a reusable object that will let you access our photon-side archiver:
+
+```python
 arch = EpicsArchive()
+```
+
+To get data, use the following:
 
-#To get data, use --> 
+```python
 data = arch.get("pv_name", xarray=True)
+```
 
-#To get the data in a data frame format --> 
-data.to_dataframe()
+It's recommended to use ``xarray=True`` in order to get an easy-to-use time series
+array back.
 
-#To get values of the data frame, such as vals or stat, for example --> 
-data.to_dataframe()['pv_name']['vals']
+To get the data as a pandas data frame, you can use:
+
+```python
+df = data.to_dataframe()
 ```
+
+To get just values (or alarm status) from the data frame, use the following:
+```python
+df = data.to_dataframe()
+values = df["pv_name"].vals
+alarm_stat = df["pv_name"].stat
+```
+
+To resample the values to just once per day, interpolating linearly (common
+alternative options may include "quadratic" or "cubic" or "nearest"):
+
+```python
+data.resample(time="1D").interpolate("linear")
+```
+
+To merge data from different PVs - using a real pair for example - one option
+may be the following:
+
+```python
+In [1]: m1 = arch.get("AT2K2:L2SI:MMS:01.RBV", xarray=True).resample(time="1D").interpolate("linear").to_dataframe()
+
+In [2]: m2 = arch.get("AT2K2:L2SI:MMS:02.RBV", xarray=True).resample(time="1D").interpolate("linear").to_dataframe()
```

Comment:
What exactly does this interpolation do to the data? I think the correct behavior is to assume a value has not changed until we get an update.

Reply:
From what I see - with very little xarray experience, admittedly - the 'resample' gives an intermediate GroupBy/DatasetResample object: http://xarray.pydata.org/en/stable/generated/xarray.Dataset.resample.html

Effectively an "OK, now what?" It's up to you to then say "OK, pick the nearest values - `.resample().nearest()`" or a "interpolate the values you're summarizing using a linear model `.resample().interpolate("linear")`". 

You could merge the xarrays, then use `ffill` and `bfill` to perform nan-filling (oddly it requires an additional Python library "bottleneck"), then resample from there, I think, too. 

I'm not sure what the "best" way is here, but the above is "a way". It'd be good to fill out even the simple README/confluence with similar/better options at some point.